File: Pharmacity/DTOs/AddToCartDto.cs
namespace Pharmacity.DTOs
{
    public class AddToCartDto
    {
        public int MaThuoc { get; set; }
        public int SoLuong { get; set; }
        public int? MaLo { get; set; } // Thêm trường này, cho phép null
    }
}

File: flutter/app512/lib/data/models/cart/add_to_cart_dto.dart
class AddToCartDto {
  final int maThuoc;
  final int soLuong;

  AddToCartDto({required this.maThuoc, required this.soLuong});

  Map<String, dynamic> toJson() => {
    'maThuoc': maThuoc,
    'soLuong': soLuong,
  };
}
